src/lib/languages/mod.rs

```diff
@@ -0,0 +1,20 @@
+use super::*;
+mod javascript;
+pub use javascript::JavaScript;
```

Comment:
I think this is a bit more organized
```suggestion
mod javascript;

use super::*;
pub use javascript::JavaScript;
```